### ISSUE TITLE
Update CachedNetworkImageProvider in didUpdateWidget

### ISSUE THREAD

**Issue by renefloor** [renefloor/flutter_cached_network_image#16]
When the url is updated, the imageprovider is not updated. A solution would be to create a new imageprovider.

```
  @override
  void didUpdateWidget(CachedNetworkImage oldWidget) {
    super.didUpdateWidget(oldWidget);
    if (widget.imageUrl != oldWidget.imageUrl ||
        widget.placeholder != widget.placeholder){

      _imageProvider = new CachedNetworkImageProvider(widget.imageUrl,
          errorListener: _imageLoadingFailed);

      _resolveImage();
    }
  }
```

**Comment by HelenGuov** [comment]
I also have the same problem, is it possible to find out when this issue would be addressed? Thanks.

**Comment by renefloor** [comment]
This should be fixed now in 0.4.0
